tutorials/04_lightmap.md

```diff
@@ -0,0 +1,86 @@
+\page lightmap Creating and using light maps
+
+This example shows how to bake light maps in Blender and apply the light map texture in Ignition.
+
+## Light maps
+
+What is baked lighting?
+
+  * Baked lighting is pre-calculated/rendered lighting information from realtime lighting. These bakes are often called light maps.
+
+Why are light maps useful?
+
+  * Light maps allow you to add global illumination, shadows, and ambient lighting at a relatively low computational cost.
+
+## Baking Light maps in Blender
+
+In Blender you can bake light map information fast and efficiently with its built in renderers. For this workflow we will be using cycles renderer.
+
+### Step 1: Create or import model for your scene
+
+To import a model go to `file` > `import` > (choose file type of model)
+
+@image html img/lightmap_import.png
+
+### Step 2: UV Mapping
+
+The next step is to create UV unwraps for your model. In Blender usually the first UVmap for any mesh created is for PBR textures and the second UV channel is for light map information. This secondary UV set is different from the UVs used to map things like color maps or normal maps because each polygon surface is receiving different lighting from all other polygons so each UV coordinate also needs to be unique.
+
+For a fast automatic light map unwrap hit F3 and then search for “Lightmap pack”
+Make sure to the “margin” is set to 1 for max distance between faces (reduces shadow bleeding during bake)
+
+@image html img/lightmap_unwrap.png
+
+### Step 3: Creating texture for model
+
+With the model selected in “Object Mode” select the “Texture Paint” tab at the top of the tool bar and create a “new texture” for the model (make sure the Alpha is checked off).
+
+This new texture should be applied to the Light Map specific UV island that was created in step 2
+
+If the texture is black use the color picker and select the UV island and it should turn white (easier to see the baked shadows).
+
+@image html img/lightmap_new_texture.png
+
+### Step 4: Material shader
+
+In order to have the texture show we must assign it to a new material. WIth your model selected open the Shader tab and select `Add` > `Image` > `Image texture`. This will create a new image texture node.
```

Comment:
```suggestion
In order to have the texture show we must assign it to a new material. With your model selected open the Shader tab and select `Add` > `Image` > `Image texture`. This will create a new image texture node.
```

Reply:
fixed. 187cbe2